Backend/Tumultu/Application/FileVariants/Commands/CreateFileVariantCommand.cs

```diff
@@ -37,6 +31,13 @@ public async Task<Guid> Handle(CreateFileVariantCommand request, CancellationTok
             File = request.File
         };
 
+        var existingVariantsByUser = (await _repository.GetAllFileVariantsByFile(request.File))
+            .Where(x => x.UploadedBy == request.UploadedBy);
+        if (existingVariantsByUser.Count() > 0)
```

Comment:
This might trigger 2 enumerations of the collection

Reply:
What's the alternative?

BTW. I've Googled a bit about the difference between .Any() and .Count(), and VS is of course bullsh*ting a little bit. .Count, as a property is indeed faster, .Count() is a slower than Any() on IEnumerables, it should run faster on other type of collections.
https://code-maze.com/any-vs-count-dotnet/
https://stackoverflow.com/questions/305092/which-method-performs-better-any-vs-count-0